CHANGELOG.md

```diff
@@ -950,7 +950,7 @@ Please see the [migration guide](doc/MIGRATION-v6.md)
 ### 5.0.0-alpha-3 (2020-12-10)
 
 #### Bugs
-* Fix #2519: Generated schemas contains a valid meta-schema URI reference (`http://json-schema.org/draft-05/schema#`)
+* Fix #2519: Generated schemas contains a valid meta-schema URI reference (`http://json-schema.org/draft-07/schema#`)
```

Comment:
Should not be changed

+ Should contain an entry for 6.5-SNAPSHOT with the change

Reply:
resolved.